counsel.el

```diff
@@ -6268,12 +6271,24 @@ The properties include:
 This variable is suitable for addition to
 `savehist-additional-variables'.")
 
+(defvar counsel-compile-history-exact-src nil
+  "Tells `counsel-compile-get-filtered-history' to exactly match the
+src file when generating it's list. This is usually set to nil except
+  for the special case of standalone files where matching what is
+  usually $HOME can cause confusion.")
```

Comment:
Please follow the Emacs docstring conventions described under [`(info "(elisp) Documentation Tips")](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html) here and elsewhere in this patch.

In particular, please keep the docstring's first summary sentence on a single line, end sentences with two spaces after the full stop, and don't indent the docstring.

Also: "it's" -> "its", and "src" -> "source", though again the docstring doesn't make it clear which "source file" is being matched in this context.